packages/framer-motion/src/animation/animators/js/index.ts

```diff
@@ -143,16 +144,22 @@ export function animateValue<V = number>({
          * a pending operation that gets resolved here.
          */
         if (speed > 0) startTime = Math.min(startTime, timestamp)
+        if (speed < 0)
+            startTime = Math.min(timestamp - totalDuration / speed, startTime)
```

Comment:
This bit is doing the same thing as the line above, to calibrate the startTime because timestamp can come in lower, but from the reverse direction.